live/global.ini

```diff
@@ -43238,7 +43238,7 @@ crusader_UGF_destroy_desc_001=Wir haben einen kürzlichen Anfall von Überdosen
 crusader_UGF_eliminateall_allies_desc_intro=Um die Sicherheit von Crusader und seiner Umgebung zu erhöhen, sucht Crusader Security qualifizierte Personen, die bei der Sicherheitsarbeit in diesem Sektor helfen.\n\nWir haben festgestellt, dass Sie über die erforderlichen Fähigkeiten verfügen, um Ihnen einen ersten Testauftrag anbieten zu können. Sie unterstützen Crusader Security im Umgang mit einer kleinen feindlichen Gruppe, die ~mission(Location|Address) bedroht hat.\n\nBitte beachten Sie, dass wir bereits Sicherheitspersonal vor Ort haben, sodass Sie im Hinblick auf Eigenbeschuss vorsichtig sein müssen.\n\nNach erfolgreichem Abschluss dieses Auftrags sind Sie berechtigt, sich für zukünftige Sicherheitsarbeiten zu bewerben sobald welche verfügbar sind.\n\nAUFTRAG AUTORISIERT DURCH:\nKontaktoffizier Bautista\nID# 948J030K\n
-crusader_UGF_eliminateall_allies_desc_rehire=Herzlichen Glückwunsch! Crusader Security hat beschlossen, Ihnen eine weitere Chance als Sicherheitsunternehmer zu bieten. Mit ein wenig harter Arbeit und Anstrengung besteht die Hoffnung, dass Sie alle bisherigen Leistungsprobleme überwinden können.\n\nZuerst möchten wir Sie jedoch bitten, Crusader Security bei der Bekämpfung einer Outlaw-Gruppe zu unterstützen, die Drohungen gegen ~mission(Location|Address) ausgesprochen hat, um Ihre Überlebensfähigkeit noch einmal zu beurteilen.\n\nDa Sie in der Vergangenheit Leistungsprobleme hatten, möchte ich Sie daran erinnern, dass Eigenbeschuss nicht toleriert wird.\n\nNach erfolgreicher Beendigung dieser Testphase wird Ihre Berechtigung wiederhergestellt und Sie können sich erneut für eine Sicherheitsarbeit bei Crusader bewerben.\n\nAUFTRAG AUTORISIERT DURCH:\nKontaktoffizier Bautista\nID# 948J030K\n
+crusader_UGF_eliminateall_allies_desc_rehire=Herzlichen Glückwunsch! Crusader Security hat beschlossen, Ihnen eine weitere Chance als Sicherheitsunternehmer zu bieten. Mit ein wenig harter Arbeit und Anstrengung besteht die Hoffnung, dass Sie alle bisherigen Leistungsprobleme überwinden können.\n\nZuerst möchten wir Sie jedoch bitten, Crusader Security bei der Bekämpfung einer Outlaw-Gruppe zu unterstützen, die Drohungen gegen ~mission(Location|Address) ausgesprochen hat, um Ihre Überlebensfähigkeit noch einmal zu beurteilen.\n\nDa Sie in der Vergangenheit Leistungsprobleme hatten, möchte ich Sie daran erinnern, dass Friendly Fire nicht toleriert wird.\n\nNach erfolgreicher Beendigung dieser Testphase wird Ihre Berechtigung wiederhergestellt und Sie können sich erneut für eine Sicherheitsarbeit bei Crusader bewerben.\n\nAUFTRAG AUTORISIERT DURCH:\nKontaktoffizier Bautista\nID# 948J030K\n
```

Comment:
"Eigenbeschuss" passt aber ziemlich gut. Friendly Fire ist ja kein Eigenname. Ich weiß wir haben das in der Festlegungsdatei. Aber Eigenbeschuss ist ein schönes Wort dafür.